src/core/reducers/categories.js

```diff
@@ -96,6 +106,47 @@ export function createEmptyCategoryList(): EmptyCategoryListType {
   }, {});
 }
 
+export const getCategories = (
```

Comment:
I don't think we need `getCategores` at all. It's not used anywhere except to generate data that is then passed into `getCategoryNames`. 

We'll need `getCategoryNames` to also accept arguments for `appName` and `addonType`, but if you do that then I believe you can add the logic of `getCategores` into `getCategoryNames` and then `AddonMoreInfo` will only have to make one call to `getCategoryNames`, and not two calls to `getCategores` and then to `getCategoryNames`.

Does that make sense?

Reply:
Yes, that make sense but I found that in `amo/components/Categories/index.js` also `getCategories` can be called  replacing the code
```
    if (
      categoriesState &&
      categoriesState[clientApp] &&
      categoriesState[clientApp][addonType]
    ) {
      categories = Object.values(categoriesState[clientApp][addonType]);
    }
```

Reply:
@bobsilverberg That's why I created that `getCategories` function. So, should I remove `getCategories` function?

Reply:
Ah, I didn't realize that because you didn't include changes to `amo/components/Categories/index.js` in this patch, but that's a good find! If that's the case, then it's probably worthwhile having. Still, you could call `getCategories` from `getCategoryNames` in the reducer, thus avoiding calling both `getCategories` and `getCategoryNames` from `AddonMoreInfo`. Also, please include the changes to `amo/components/Categories/index.js` in this patch.

Reply:
Yes sure! I will change that too and start working on the tests.